features/storage/csi_clone.feature

```diff
@@ -0,0 +1,67 @@
+Feature: CSI clone testing related feature
+  """
+  This test can only work on a Cluster which has CSI driver deployed, OSP has csi deploy by default from 4.7. 
+  When deploy a cluster which has no csi deployed, please use enable_tpnu_featureset: "yes" in LAUNCHER_VARS.
+  From 4.7, OSP CSI driver was installed by default. Its storage class name is standard-csi.
+  Use below command can temporarily add cs class(oc get sc, can find all deployed stroage class):
+  oc patch featuregate cluster -p '{"spec": {"featureSet": "TechPreviewNoUpgrade"}}' --type merge
+  """
+  
+  # @author jianl@redhat.com
+  # @case_id OCP-27615
+  @admin
+  Scenario: Clone a pvc on OSP in the same namespace is successfully
+    Given the master version >= "4.8"
+    Given I switch to cluster admin pseudo user
+    # Step 1
+    # From 4.7, OSP CSI driver was installed by default. The storage class name is standard-csi
+    # Step 2
+    When admin creates a project with:
+      | project_name | clone |
+    Then the step should succeed
+    Given I use the "clone" project
+    When I obtain test data file "storage/csi/pod_pvc_sc.yaml"
```

Comment:
To avoid maintaining too many yaml files, we can use some existing one if it matches your requirement.
Like storage/misc/pvc.json and storage/misc/pod.yaml, only add one pvc yaml file for cloned pvc.

Reply:
@duanwei33 I don't want to share files with others, because I have experience that someone in a team changes a public file, it blocked the others...